### PR TITLE
Fixes Issue #30 | Fig 2.23 | centralLimitDemo.ipynb | Book 1 

### PR DESCRIPTION
**Pull request by AnkitaKumariJain14**
### Figure Number

Book 1 - 2.23 

### Figures

[gist](https://gist.github.com/AnkitaKumariJain14/65af3847d59db9f1f245c0c4e24e94b0)

### Issue 

[#30](https://github.com/pyprobml-review/pyprobml-review/issues/30)

## Steps

- [x] Changed the name of the variables/functions to PEP 8 style
- [x] Added code to generate two figures as given in the book and Latexified them.
- [x] Converted code to JAX
- [x] Added interactive demo to change number of Random Variables and observe the sample mean.

 
## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

## Potential problems/Important remarks
- The figures generated with Key =1 vary from the original figures in the book. The initial numpy code also generates figures that differ from the book. 
cc: @karm-patel @patel-zeel

### CODE REVIEW

File: notebooks/book1/02/centralLimitDemo.ipynb [hunk 1]
```diff
@@ -0,0 +1,653 @@
+{
```
Comment (karm-patel) [inline notebooks/book1/02/centralLimitDemo.ipynb:1]: <p>Currently this function looks very ambigous, I had a discussion with zeel regarding this. we should change such that it can be interpretable easily. </p><p>a) <code>bins</code> argument is useless here b) N should be a number, making it as a array is confusing.  </p><p>We want to do following step:</p><ol><li>Take N samples from Beta and sum over it, it will give us single mean. make a function for the same.</li><li>Repeat step-1 sampleSize (10000) times using <code>jax.vmap</code>  and this will return means. make a function for the same. </li><li>Then plot histogram over means using <code>plot_convolutionHist()</code> but again take N as single number, not the array </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (karm-patel) [inline notebooks/book1/02/centralLimitDemo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.            plt.ylabel("$mean$")
</pre><p>For xlabel and ylabel, we sould search about this, I think at x_label we can use "bins".  And for y-label I'm not sure we can use "mean" or something different word given that this is histogram plot not the pdf plot. One way is to check any histogram plot in mml book.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (AnkitaKumariJain14) [reply notebooks/book1/02/centralLimitDemo.ipynb:1]: The file has been updated. Please have a look and let me know

File: notebooks/book1/02/centralLimitDemo.ipynb [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
```
Comment (karm-patel) [inline notebooks/book1/02/centralLimitDemo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=2, fig_height=2)
</pre><p>please set <code>fig_height=1.85</code> I checked in latex source code </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Comment by karm-patel** [comment]
Hi @AnkitaKumariJain14, Sorry for the delay, after changing this `fig_height` you can go ahead to create a PR on main repo.

File: notebooks/book1/02/centralLimitDemo.ipynb (hunk 2 above)
Comment (AnkitaKumariJain14) [reply notebooks/book1/02/centralLimitDemo.ipynb:1]: Has been updated. Will submit a PR now
